src/index.js

```diff
@@ -14,16 +15,19 @@ export default function (userCss, ...values) {
     css = csjs(css, ...values);
   }
 
+  let cssText = csjs.getCss(css);
+  const scope = scopedName(cssText)('DecoratedComponent');
```

Comment:
That would only work in development, in production Uglify takes that away unfortunately.